views/update/overseas-entity-query.html

```diff
@@ -12,15 +12,30 @@
 {% endblock %}
 
 {% block content %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
 
         <h1 class="govuk-heading-xl">{{title}}</h1>
+        {% if errorList %}
+            {{ govukErrorSummary
+              ({
+                titleText: "There is a problem",
+                errorList: [
+                {
+                  text: errorList,
+                  href: "#oe_number"
+                }
+               ]
+              }) 
+            }}
+        {% endif %}
         {% include "includes/list/errors.html" %}
         <form method="post">
-
```

Comment:
I think this change can also be reverted as changes for validation has been merged

Reply:
Please elaborate as I didn't follow the validation changes

Reply:
Did you make any changes to this file, as there were some changes in the validation ticket and thought it was just highlighting that, if you made changes to the file then it seems fine